src/amo/components/InstallWarning/index.js

```diff
@@ -84,10 +86,16 @@ export class InstallWarningBase extends React.Component<InternalProps> {
   };
 
   componentDidMount() {
-    const { _log, _tracking, clientApp, variant } = this.props;
+    const {
+      _log,
+      _tracking,
+      clientApp,
+      isUserInExperiment,
+      variant,
+    } = this.props;
 
-    if (!variant) {
-      _log.debug(`No variant set for experiment "${EXPERIMENT_ID}"`);
+    if (!variant || !isUserInExperiment) {
+      _log.debug(`User not enrolled in experiment "${EXPERIMENT_ID}"`);
```

Comment:
Isn't `!variant` different than not being enrolled in the experiment?

Reply:
Ok, I'll update it to do that.